esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
+{% autoescape off %}
+{{ msgbdy }}
```

Comment:
If `msgbdy` might have HTML, you'll need to use `{{ msgbdy|safe }}` so django won't escape the HTML.  (When doing this you should make sure that only admins are passing `msgbdy`, or that it's escaped somewhere else; in this case the former is true.)